std_msgs/String ns
std_msgs/String controller_name
---
std_msgs/Bool answer